src/css.js

```diff
@@ -336,16 +333,23 @@ jQuery.each( [ "height", "width" ], function( i, name ) {
 		},
 
 		set: function( elem, value, extra ) {
-			var styles = extra && getStyles( elem );
-			return setPositiveNumber( elem, value, extra ?
-				augmentWidthOrHeight(
+			var matches = rcssNum.exec( value ),
```

Comment:
Verification works; if it were me you know I'd go with whichever yields a smaller gzip. And `matches` is fine here... I meant for that to be a comment about the `setPositiveNumber` parameter name, but I guess it's made moot by the other reasons to remove it.